examples_test/utils.go

```diff
@@ -0,0 +1,50 @@
+package examples
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+func randNamef(f string) string {
```

Comment:
```
examples_test/utils.go:16:6: `randName` is unused (deadcode)
func randName() string {
     ^
examples_test/utils.go:20:6: `newClient` is unused (deadcode)
func newClient(token string) (*aiven.Client, error) {
     ^
examples_test/utils.go:37:6: `newTFConfig` is unused (deadcode)
func newTFConfig(providerPath string) (string, error) {
     ^
examples_test/utils.go:12:6: func `randNamef` is unused (unused)
func randNamef(f string) string {
```

Reply:
Do we need this file?

Reply:
> Let's check if the AIVEN_TOKEN and AIVEN_PROJECT_NAME environment vars are set and, if not, error with the meaningful message before running the tests. WDYT?

There is a test for that.

> When we run example tests, can we add some prefixes to a service name? Something like ex-test? 

I can do that.

> Do we need this file?

That file is used by the test. But test marked with tag and the file is not. I'll mark them both, hope that will help.